recipes/mg5amcnlo/meta.yaml

```diff
@@ -0,0 +1,160 @@
+    - {{ stdlib('c') }}
+    # Compilers needed to set names in mg5_configuration.txt and will be
+    # used in the future along with make when bugs with Pythia8 are fixed
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    # - make
+    - sed
+    - grep
+  host:
+    - python
+    - six
+    - fastjet-cxx
+    - emela
+    - lhapdf
+  run:
+    # mg5amcnlo is a code generator and so requires a build system at runtime
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - tar
+    - perl
+    - python
+    - six
+    - numpy
+    - fastjet-contrib
+    - emela
+    - lhapdf
+    - pythia8
+    - ply
+    - oneloop
+    - ninja-hep-ph
+    - collier
+    - ghostscript
+
+test:
+  files:
+    - $RECIPE_DIR/tutorial.mg5
+  commands:
+    - test -d $PREFIX/MG5_aMC
+    - test -d $PREFIX/MG5_aMC/aloha
+    - test -d $PREFIX/MG5_aMC/bin
+    - test -d $PREFIX/MG5_aMC/HELAS
+    - test -d $PREFIX/MG5_aMC/input
+    - test -d $PREFIX/MG5_aMC/madgraph
+    - test -d $PREFIX/MG5_aMC/MadSpin
+    - test -d $PREFIX/MG5_aMC/mg5decay
+    - test -d $PREFIX/MG5_aMC/models
+    - test -d $PREFIX/MG5_aMC/PLUGIN
+    - test -d $PREFIX/MG5_aMC/Template
+    - test -d $PREFIX/MG5_aMC/vendor
+
+    - test -f $PREFIX/MG5_aMC/proc_card.dat
+    - test -f $PREFIX/MG5_aMC/VERSION
+    - test -f $PREFIX/bin/mg5_aMC
+    - test -f $PREFIX/MG5_aMC/input/mg5_configuration.txt
+    - test -f $PREFIX/MG5_aMC/madgraph/LICENSE
+
+    - echo -e "\n# Checking mg5_configuration.txt"
+    - cat $PREFIX/MG5_aMC/input/mg5_configuration.txt
+
+    - mg5_aMC --help
+
+    - echo "exit" | mg5_aMC
+
+    - mg5_aMC $RECIPE_DIR/tutorial.mg5
+
+about:
+  home: https://github.com/mg5amcnlo/mg5amcnlo
+  summary: 'MadGraph5_aMC@NLO'
+  description: |
+    MadGraph5_aMC@NLO is a framework that aims at providing all the elements
+    necessary for SM and BSM phenomenology, such as the computations of cross
+    sections, the generation of hard events and their matching with event
+    generators, and the use of a variety of tools relevant to event manipulation
+    and analysis.
+    Processes can be simulated to LO accuracy for any user-defined Lagrangian,
+    and the NLO accuracy in the case of models that support this kind of calculations
+    &mdash; prominent among these are QCD and EW corrections to SM processes.
+    Matrix elements at the tree- and one-loop-level can also be obtained.
+
+    MadGraph5_aMC@NLO is the new version of both MadGraph5 and aMC@NLO that unifies
+    the LO and NLO lines of development of automated tools within the MadGraph family.
+    As such, the code allows one to simulate processes in virtually all configurations
+    of interest, in particular for hadronic and e+e- colliders; starting from version
+    3.2.0, the latter include Initial State Radiation and beamstrahlung effects.
+  # modified University of Illinois/NCSA license
+  license: LicenseRef-NCSA
+  license_family: OTHER
+  license_file: source/LICENSE
```

Comment:
License is here: https://github.com/mg5amcnlo/mg5amcnlo/blob/3d6799de1bcabee5173e8a3711f657a5cd5940cf/LICENSE